ydb/core/driver_lib/run/kikimr_services_initializers.cpp

```diff
@@ -824,10 +825,14 @@ void TBasicServicesInitializer::InitializeServices(NActors::TActorSystemSetup* s
 
     if (Config.HasTracingConfig()) {
         const auto& tracing = Config.GetTracingConfig();
+        std::unique_ptr<NWilson::IGrpcSigner> grpcSigner;
+        if (Factories && Factories->WilsonGrpcSignerFactory) {
+            grpcSigner = Factories->WilsonGrpcSignerFactory();
+        }
+        auto wilsonUploader = NWilson::CreateWilsonUploader(tracing.GetHost(), tracing.GetPort(), tracing.GetRootCA(), tracing.GetServiceName(), std::move(grpcSigner));
```

Comment:
just for the better readability I would recommend to change arguments to struct. maybe not in this commit, but later

Reply:
Same as [here](https://github.com/ydb-platform/ydb/pull/768#discussion_r1438462802)